README.md

```diff
@@ -45,5 +45,25 @@ gulp test/dotnet
 2. Next, implement the ```run``` method under the rule that actually does the validation. Add a reference to this script file under ```src/typescript/azure-openapi-validator/index.ts```.
 3. Lastly add a test case for the validation rule, by adding a test json/yaml under ```src/typescript/azure-openapi-validator/tests/resources``` and a script under ```src/typescript/azure-openapi-validator/tests``` depending on the type of the validation rule.
 
+## Changelog

```

Comment:
We should probably have a separate changelog, I agree, if we wanted to put what's new in the readme for every release,  I guess we could point of copy/paste to the latest date here, but probably don't want to keep history in the readme.